src/config.js

```diff
@@ -1,6 +1,5 @@
 // get openapi schema
-import { API } from './src/api.js'
-import { dev } from './wrangler.json'
+import { API } from './api.js'
 
 // the packages to proxy
 export const PROXIES = []
```

Comment:
can probably move most of these settings to the dmno config eventually - then you can get validations and docs. seemed like mostly these weren't doing anything yet, so I jsut left them for now.